updateHostsWindows.bat

```diff
@@ -20,11 +20,25 @@ if not exist "%WINDIR%\System32\drivers\etc\hosts.skel" (
 	COPY %WINDIR%\System32\drivers\etc\hosts %WINDIR%\System32\drivers\etc\hosts.skel
 )
 
+:: Make sure we are in the correct working directory
+rem Path info from:
+rem https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call#batch-parameters
+cd %~dp0
+
 :: Update hosts file
-python updateHostsFile.py -a
+if exist updateHostsFile.exe (
+    echo run exe

```

Comment:
Do we really need the echo?

Reply:
Nope, I just hadn't used the `if` syntax in a batch file before and wanted to test both branches without a full run.

Reply:
Oh, and nothing personal. :) Everyone is just trying to make the best tools we can.

Reply:
> Nope, I just hadn't used the `if` syntax in a batch file before and wanted to test both branches without a full run.

Great, you can get rid of it then :wink:

Reply:
> Oh, and nothing personal. :) Everyone is just trying to make the best tools we can.

:+1: